tests/package.test.js

```diff
@@ -0,0 +1,13 @@
+import expect from 'expect';
+
+const pkg = require('../dist/microstates.es.js');
```

Comment:
Could we not just make these assertions on the index?

Reply:
We can, but I really want a smoke test to ensure that the built assets have the export.

Reply:
I changed this as well.